SECURITY.md

```diff
@@ -0,0 +1,57 @@
+# Security Policy
+
+## Supported Releases
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.0.1   | :white_check_mark: |
+| 2.0.0   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+Please do not open public github issues or pull requests on this repository for security-related matters. Instead, follow the responsible disclosure process outlined below.
+
+### Scope
+This security policy covers the security of this repository and its code. If you discover a security vulnerability related to this repository, please report it responsibly.
+
+### Responsible Disclosure Process
+#### Option 1: Reporting directly via Github
+
+1. **Contact Us**: Contact [@salahlalami](https://github.com/salahlalami) with details of the vulnerability. Please provide a clear and concise description of the issue, any potential impact, and a step-by-step demonstration if possible. Please do not include sensitive information in your initial email.
```

Comment:
@vvxhid add this email security@idurarapp.com

Reply:
Done.